src/o365/spo/SpoCommand.ts

```diff
@@ -124,4 +127,110 @@ export default abstract class SpoCommand extends Command {
         });
     });
   }
+
+  public ensureFormDigest(cmd: CommandInstance, context: FormDigestInfo, debug: boolean): Promise<FormDigestInfo> {
+    return new Promise<FormDigestInfo>((reject: (error: any) => void): void => {
```

Comment:
The first argument of the new Promise should be the resolve not the reject callback. Compare with the original implementation to see the difference.